parser.go

```diff
@@ -32,7 +40,7 @@ func ParseMetric(packet []byte) (*Metric, error) {
 	}
 
 	// Create a new hasher for accumulating the data we read
-	h := fnv.New32()
+	h := fnv.New32a()
```

Comment:
Nope, that’s fine by me. I hadn’t come across the recommendation for 1a over 1. :+1: